docs/output.md

```diff
@@ -239,3 +183,71 @@ The pipeline has special steps which allow the software versions used to be repo
   * Directory containing parsed statistics from the different tools used in the pipeline
 
 For more information about how to use MultiQC reports, see [http://multiqc.info](http://multiqc.info)
```

Comment:
Not sure about that link ;-)

Reply:
You mean https?

Reply:
Aaaaah, sorry, I believed we were in the fusion-report section...